modules/tinymce/src/plugins/table/test/ts/browser/TableRemoveTrailingBrTest.ts

```diff
@@ -9,13 +9,16 @@ import * as TableTestUtils from '../module/test/TableTestUtils';
 describe('browser.tinymce.plugins.table.TableRemoveTrailingBrTest', () => {
   const baseSettings = {
     plugins: 'table code',
-    base_url: '/project/tinymce/js/tinymce'
+    base_url: '/project/tinymce/js/tinymce',
+    table_default_attributes: { },
+    table_default_styles: { width: '200px' },
```

Comment:
What is going on here exactly. This doesn't seem to have anything to do with the test perf so we are not reverting something here are we?